ostd/Cargo.toml

```diff
@@ -2,41 +2,53 @@
 name = "ostd"
 version = "0.6.2"
 edition = "2021"
+description = "Rust OS framework that facilitates the development of and innovation in OS kernels"
+license = "MPL-2.0"
+readme = "README.md"
+repository = "https://github.com/asterinas/asterinas"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
+# Settings for publishing docs in docs.rs
+[package.metadata.docs.rs]
+default-target = "x86_64-unknown-none"
+targets = ["x86_64-unknown-none"]
+
+# This is a temporary fix for the lint warnings about unknown config `ktest`.
+# FIXME: this lint may be removed in the future.
+[lints.rust]
+unexpected_cfgs = { level = "warn", check-cfg = ['cfg(ktest)'] }
```

Comment:
It can be removed since #864 has added `cfg(ktest)` to known-cfg in OSDK building.